config/schemas/config.yaml

```diff
@@ -0,0 +1,2854 @@
+              absoluteLifetime:
+                default: 3960h
+                type: string
+              reuseInterval:
+                default: 3s
+                type: string
+              validIfNotUsedFor:
+                default: 2160h
+                type: string
+            type: object
+          signingKeys:
+            default: 6h
+            type: string
+        type: object
+      google:
+        properties:
+          groupSupport:
+            default: false
+            type: boolean
+        type: object
+      nodeSelector:
+        $ref: '#/$defs/nodeSelector'
+      replicaCount:
+        default: 2
+        type: number
+      resources:
+        $ref: '#/$defs/resources'
+      serviceMonitor:
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      subdomain:
+        default: dex
+        type: string
+      tolerations:
+        $ref: '#/$defs/tolerations'
+      topologySpreadConstraints:
+        items:
+          properties:
+            labelSelector:
+              properties:
+                matchLabels:
+                  properties:
+                    app.kubernetes.io/name:
+                      default: dex
+                      type: string
+                  type: object
+              type: object
+            maxSkew:
+              default: 1
+              type: number
+            topologyKey:
+              default: kubernetes.io/hostname
+              type: string
+            whenUnsatisfiable:
+              default: DoNotSchedule
+              type: string
+          type: object
+        type: array
+    type: object
+  externalTrafficPolicy:
+    properties:
+      local:
+        default: false
+        type: boolean
+      whitelistRange:
+        properties:
+          global:
+            default: 0.0.0.0/0
+            type: string
+          dex:
+            type: boolean
+          harbor:
+            type: boolean
+          opensearch:
+            type: boolean
+          opensearchDashboards:
+            type: boolean
+          opsGrafana:
+            type: boolean
+          thanosReceiver:
+            type: boolean
+          userGrafana:
+            type: boolean
+        additionalProperties:
+          type: boolean
+        type: object
+    type: object
+  falco:
+    properties:
+      affinity:
+        $ref: '#/$defs/affinity'
+      alerts:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+          hostPort:
+            default: http://alertmanager-operated.monitoring:9093
+            format: uri
+            type: string
+          priority:
+            default: notice
+            type: string
+          type:
+            default: alertmanager
+            type: string
+        type: object
+      customRules:
+        type: object
+        additionalProperties:
+          type: string
+          format: yaml
```

Comment:
What, can one just specify that a string field contains a certain format? :eyes:

Reply:
[Sure!](https://json-schema.org/draft/2020-12/json-schema-validation#name-vocabularies-for-semantic-c) Whether the validator supports a particular format is another question... If not at least it serves as documentation 😄